src/containers/profile/ProfileContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+import { getProfileAction } from "../../actions/profileActions";
+
+import ProfileComponent from "../../components/userProfile/ProfileComponent";
+
+export class ProfileContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      username: "",
+      image: "",
+      bio: "",
+      full_name: "",
+      isUploading: ""
+    };
+  }
+
+  componentWillMount() {
+    this.props.getProfileAction();
```

Comment:
go through the file and destructure where necessary

Reply:
Done